packages/flutter_tools/templates/module/android/library_new_embedding/Flutter.tmpl/build.gradle.tmpl

```diff
@@ -30,6 +30,20 @@ group = "{{androidIdentifier}}"
 version = "1.0"
 
 android {
+    publishing {
+        singleVariant("debug") {
```

Comment:
My gut says we should either use multiVariant or setup only release to publish and not this version where we use single variant multiple times. 
https://developer.android.com/reference/tools/gradle-api/8.3/null/com/android/build/api/dsl/LibraryPublishing

Reply:
Resolving as this is now only relevant to follow up pr